### ISSUE TITLE
Soundminer / Metadata Integration

### ISSUE THREAD

**Issue by mentalsanityoff** [MTG/freesound#1563]
Submitting here via recommendation from Frederic: a personal suggestion regarding the facilitation of embedding metadata into WAV files downloaded from Freesound.org, specifically using the popular sound library software Soundminer v5Pro, to generate more searchable BWAV files.

When a user uploads a sound to Freesound.org, they are queried for various pieces of metadata, currently only searchable online at Freesound.org. My suggestion, is to make available CSV or tab delimited TXT files, pulled from a much (!) larger background database of all metadata tagged to various sounds on Freesound (surely that exists or can exist, right?). Provided the option to download this metadata, a user could use various features of Soundminer V5Pro to embed and generate metadata-rich Broadcast WAVE files.

This would avoid a potentially massive (unfeasible) re-encoding effort of all the sounds on Freesound.org with their metadata (as entered on Freesound) as BWAV's to be remotely hosted. I'd imagine establishment of such a workflow being very popular in the professional sound community, where Soundminer is ubiquitous. Can't speak to Basehead, etc.'s functionality, but know that the "Direct Import" feature within Soundminer does exactly this: https://info.soundminer.com/docs/import-text

Frederic notes that Freesound already does file generation when downloading sounds as packs (archive compression), and naturally generation of these text files is probably completely negligible in terms of storage, memory, bandwidth, etc.

Further questions can be relayed via DM at Freesound.org, or reference the original forum post below:

https://freesound.org/people/SpliceSound/

https://freesound.org/forum/bug-reports-errors-and-feature-requests/43036/?page=1#post97183

**Comment by ffont** [comment]
Hi @mentalsanityoff , thanks for opening the issue. Just to clarify, your suggestion is that when a pack is downloaded, besides the audio files and the license file a new CSV file is added that contains the metadata in a format that soundminer (and potentially other tools) can understand?

For individual sounds, would you suggest also adding some option to download a single-row+header CSV file to be loaded in soundminer and the like?

**Comment by mentalsanityoff** [comment]
Correct, single-row + header CSV files for individual sounds, so that the functionality exists not just for packs, but all sounds on Freesound.org. But the real power lies in packs, since you'd be able to tag and embed many sound files all at once.
